docs/setup_instructions/windows_installation_instructions.md

```diff
@@ -139,6 +139,19 @@ The downloaded installer will have a name like `RStudio-2022.02.2-win.exe` (the
 Open this file and follow the prompts to install RStudio onto your computer.
 You may need to give permission to modify your system.
 
+##### Update an RStudio setting
+
+Next, we recommend that you update this setting in RStudio to ensure that code we write is compatibility across different machines.
+
+In RStudio, go to `Tools > Global Options` .
+Click `Code` on the left-side menu, and select the `Saving` tab.
+
+On this screen, ensure that "None" is selected from the "Line ending conversion" dropdown.
```

Comment:
Should we say "or Posix"? I can't decide which is better default here... None seems pretty safe, but I actually don't know what will happen with new files, which could be an issue at times.

Reply:
It's also tricky since this isn't a project-specific setting, but a global RStudio one...

Reply:
I feel like posix is the better default overall, but I don't love that we are telling them to modify other files...

Reply:
What if we exclude this from setup, but add to the workshop itself in instructor notes so that this only applies to workshop files? When they open the RRP project, we can direct them to modify the project settings to use Posix. This will also give us a moment to contextualize why we are taking this step.

Reply:
I think that may make sense. Line endings are enough of a bugbear that they seem worth adding to explicit instruction. (We can also set this to posix in the exisiting project files, just to avoid the issue if we forget about it).

Reply:
> We can also set this to posix in the existing project files, just to avoid the issue if we forget about it

good call


> bugbear

I am wholly unfamiliar with this term, but the internet tells me it's DnD. Personally I like it because it's a tardigrade.